### PR TITLE
ARMBANDS

### PR DESCRIPTION
**Pull request by Bonomedia**
## About The Pull Request

Adds in funny armbands

## Why It's Good For The Game

Allows for identification between gangs, raider groups, etc

## Changelog

:cl:
add: Adds red, blue, green, purple armbands
/:cl:

### CODE REVIEW

**Comment by Bonomedia** [comment]
i thought i would be able to edit the description

**Comment by Sinbest** [comment]
a red armband....

i wonder if there are any historical organizations I could make a reference to here

File: mojave/items/clothing/other/ms13accessories.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	icon_state ='squad_blue'
+
+/obj/item/clothing/accessory/armband/ms13/green
+	name = "green armband"
+	desc = "A green armband, usually used for identification among groups."
+	icon_state ='squad_green'
+
+/obj/item/clothing/accessory/armband/ms13/purple
+	name = "purple armband"
+	desc = "A red armband, usually used for identification among groups."
```
Comment (Hekzder) [inline mojave/items/clothing/other/ms13accessories.dm]:
```suggestion
	desc = "A purple armband, usually used for identification among groups."
```

Comment (Bonomedia) [reply mojave/items/clothing/other/ms13accessories.dm]: FIXED...

**Pull request merged**